### PR TITLE
[ISSUE-474] Make functions to get global variables

### PR DESCRIPTION
**Pull request by aosandy**
### Issue dell/csi-baremetal#474

Make functions to get global variables according to the report from linter with enabled gochecknoglobals

## PR checklist
- [ ] Add link to the issue
- [ ] Choose Project
- [ ] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Testing details_

### CODE REVIEW

File: pkg/feature/security_verifier/models/components.go [hunk 1]
```diff
@@ -9,3 +9,11 @@ var (
 	// Scheduler represents the scheduler extender component
 	Scheduler Component = "Scheduler"
```
Comment (safronovD) [inline pkg/feature/security_verifier/models/components.go:10]: Could we make it constant?

File: pkg/validator/models/rbac.go [hunk 2]
```diff
@@ -11,3 +11,7 @@ type RBACRules struct {
 	Data interface{}
 	Type Rule
 }
+
+func GetServiceAccountIsRoleBound() Rule {
```
Comment (safronovD) [inline pkg/validator/models/rbac.go:15]: Could we make it constant?

File: pkg/nodeoperations/controller.go [hunk 3]
```diff
@@ -56,6 +56,14 @@ var (
 	}
 )
 
+func getMTaint() corev1.Taint {
+	return mTaint
```
Comment (safronovD) [inline pkg/nodeoperations/controller.go:60]: `mTaint` is not a global var. Do we really need it?

File: pkg/validator/models/rbac.go (hunk 2 above)
Comment (aosandy) [reply pkg/validator/models/rbac.go:15]: If we want to make ServiceAccountIsRoleBound constant, we need to initialize it with some value, but I don't know which one

File: pkg/nodeoperations/controller.go (hunk 3 above)
Comment (aosandy) [reply pkg/nodeoperations/controller.go:60]: Should I remove `rTaint` too?

File: pkg/feature/security_verifier/models/components.go (hunk 1 above)
Comment (aosandy) [reply pkg/feature/security_verifier/models/components.go:10]: Should I make `Node` constant too?

Comment (safronovD) [reply pkg/feature/security_verifier/models/components.go]: Yes, they are looking similar

File: pkg/nodeoperations/controller.go (hunk 3 above)
Comment (safronovD) [reply pkg/nodeoperations/controller.go]: Yes

**Comment by safronovD** [comment]
@aosandy Please fill PR checklist and testing strategy. Also it's need to add passed CI tests. Example: https://github.com/dell/csi-baremetal/pull/824

File: pkg/validator/models/rbac.go (hunk 2 above)
Comment (Aber4Nod) [reply pkg/validator/models/rbac.go:16]: Yes, it should be constant. Initialize it for example like this:
`var ServiceAccountIsRoleBound Rule = "ServiceAccountIsRoleBound"`

**Review by yurkov-anton — APPROVED** [review]

**Review by rkiselenko — APPROVED** [review]

**Comment by safronovD** [comment]
@aosandy please make sure that linting stage passed

**Review by CraneShiEMC — APPROVED** [review]

**Pull request closed without merging**